### PR TITLE
back button implemented

### PR DESCRIPTION
**Pull request by briansetzesglobant**
Back button implementation done

Video:
https://user-images.githubusercontent.com/84338449/129037348-277981d5-6a7e-49a6-895f-d5a444683ef2.mp4

### CODE REVIEW

File: lib/src/utils/ui_constants.dart [hunk 1]
```diff
@@ -45,4 +45,12 @@ abstract class UiConstants {
   static const double movieTitleFontSize = 30.0;
   static const double releaseDateFontSize = 20.0;
   static const double voteAverageFontSize = 36.0;
+
+  // back_button
+  static const double paddingBackButtonDetails = 10.0;
+
```
Comment (HoracioDuca) [inline lib/src/utils/ui_constants.dart]: this empty line could be removed to keep consistency

**Review by HoracioDuca — APPROVED** [review]
LGTM, just 1 minor comment

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

File: lib/src/utils/ui_constants.dart (hunk 1 above)
Comment (briansetzesglobant) [reply lib/src/utils/ui_constants.dart]: Done

**Review by Kevin-Guillen — APPROVED** [review]
LGTM

**Pull request merged**